### DISCUSSION TITLE
Weekly metrics

### DISCUSSION THREAD

**Discussion by sldblog** [ministryofjustice/hmpps-interventions-service#894] [General]
What should we measure in the `#interventions` channel
every **Monday**, for the **previous week**?

For context, I'm hoping we can find a way to track the last week's value with a graph over the past few weeks (so a trend is visible)

Ideas:

### Delivery metrics

- number of releases to production? (sum of all components)
- uptime?

### Performance (business) metrics

What is the most important metric for our team?

- number of successfully completed referrals?
- number of unsuccessfully concluded referrals?
- number of new live referrals?
- number of re-referrals?
- time of referral from start to end?
- number of drafts per probation user?

### Operational metrics

- number of support tickets?

**Comment by tomsmyers** [comment]
i have been trying to put together a query which indicates something around duplicates. i hit a stumbling block which is that the version of kibana we are running does not support [runtime fields](https://www.elastic.co/guide/en/elasticsearch/reference/8.1/runtime-examples.html), which we need to be able to extract the required contract types and crns. but if we could get this working i think it would be good. the business seems very interested in this data point right now.

**Reply by tomsmyers** [reply]
you can do something really basic in app insights which shows you "number of times someone was referred twice (for any service) in a given time bucket", but i'm not sure it really has the detail we need.

**Reply by sldblog** [reply]
is that similar to something like 're-referrals last week'?
